homeassistant/components/hue/switch.py

```diff
@@ -22,9 +22,9 @@
 from .const import DOMAIN
 from .v2.entity import HueBaseEntity
 
-ControllerType = Union[LightLevelController, MotionController]
+ControllerType: TypeAlias = LightLevelController | MotionController
 
-SensingService = Union[LightLevel, Motion]
+SensingService: TypeAlias = LightLevel | Motion
```

Comment:
Sometimes with the new syntax, mypy needs a little help to know that it should be a type. Adding `TypeAlias` usually helps in these cases.